### ISSUE TITLE
Improve error messages when server is misconfigured

### ISSUE THREAD

**Issue by tasn** [svix/svix-webhooks#1435]
E.g. try not setting `jwt_secret` and see how confusing the errors are.

**Comment by svix-jplatte** [comment]
With https://github.com/svix/svix-webhooks/pull/1520 merged, I think what's left here is:

- retry schedule
- database config

which produce bad error messages due to `#[serde(untagged)]`.

**Comment by svix-jplatte** [comment]
Yes, nothing has happened with regard to this since the last update I posted in November.
